sql/catalyst/src/test/scala/org/apache/spark/sql/HashBenchmark.scala

```diff
@@ -87,18 +94,39 @@ object HashBenchmark {
       .add("binary", BinaryType)
       .add("date", DateType)
       .add("timestamp", TimestampType)
-    test("normal", normal, 128)
+    /*
+    Intel(R) Core(TM) i7-4960HQ CPU @ 2.60GHz
+    Hash For normal:                   Avg Time(ms)    Avg Rate(M/s)  Relative Rate
+    -------------------------------------------------------------------------------
+    interpreted version                     2187.63             0.96         1.00 X
+    codegen version                         1693.21             1.24         1.29 X
+     */
+    test("normal", normal, 1024 * 4)
 
     val arrayOfInt = ArrayType(IntegerType)
     val array = new StructType()
       .add("array", arrayOfInt)
       .add("arrayOfArray", ArrayType(arrayOfInt))
-    test("array", array, 64)
+    /*
+    Intel(R) Core(TM) i7-4960HQ CPU @ 2.60GHz
+    Hash For array:                    Avg Time(ms)    Avg Rate(M/s)  Relative Rate
+    -------------------------------------------------------------------------------
+    interpreted version                     3290.06             0.08         1.00 X
+    codegen version                         6674.07             0.04         0.49 X
+     */
+    test("array", array, 512)
 
     val mapOfInt = MapType(IntegerType, IntegerType)
     val map = new StructType()
       .add("map", mapOfInt)
       .add("mapOfMap", MapType(IntegerType, mapOfInt))
-    test("map", map, 64)
+    /*
+    Intel(R) Core(TM) i7-4960HQ CPU @ 2.60GHz
+    Hash For map:                      Avg Time(ms)    Avg Rate(M/s)  Relative Rate
+    -------------------------------------------------------------------------------
+    interpreted version                    64709.73             0.00         1.00 X
```

Comment:
How long does this benchmark take to run? THis looks really long. I think we should keep benchmarks to run in low number of seconds total if possible.